pennlabs/shortener/manager.py

```diff
@@ -0,0 +1,24 @@
+import hashlib
+
+from django.db import models
+
+
+class UrlManager(models.Manager):
+    def get_or_create(self, long_url):
+        if self.filter(
+            long_url=long_url
+        ).exists():  # If a shortened URL already exists, don't make a duplicate.
+            return self.get(long_url=long_url), False
+
+        try:
+            hashed = hashlib.sha3_256(long_url.encode("utf-8")).hexdigest()
```

Comment:
So it looks like this was added in python 3.6. Since we only support 3.6+ we can remove the try/catch and just assume this call will work